src/userData.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const baseAxios = axios.create({
+  baseURL: "https://bootcamp-api.codeit.kr/api/",
+});
+
+const getUserData = async () => {
+  try {
+    const userData = await baseAxios.get("sample/user");
+    return userData;
+  } catch (error) {
+    console.error("Error fetching user data:", error);
+  }
```

Comment:
에러시 console만 찍기보다 사용처에서 에러를 핸들링 할 수 있도록 전달해주면 좋을 것 같아요.
```suggestion
  } catch (error) {
    console.error("Error fetching user data:", error);
    throw error;
  }
```